controllers/apiController.js

```diff
@@ -0,0 +1,40 @@
+function index(req, res) {
+  res.json({
+    message: "Welcome to dogShopz!",
+    documentation_url: "https://agile-wildwood-44374.herokuapp.com/",
+    base_url: "localhost:3000",
+    endpoints: [
+      {
+        method: "GET", path: "/api", description: "Describes available endpoints"
+      }
+    ]
+  });
+}
+
+
+
+
+ module.exports = {
+  index: index
+}
+
+
+//  app.get('/api', function apiIndex(req, res) {
```

Comment:
File looks good!
Once a file is complete, remove  unused pseudocode and delete extra spaces.